src/llmcompressor/pipelines/sequential/helpers.py

```diff
@@ -310,11 +313,13 @@ def partition_graph(model: Module, partitions: List[List[Node]]) -> List[Subgrap
         # save the subgraph for this partition
         graph.lint()
         input_names = set(node.name for node in graph.nodes if node.op == "placeholder")
+        modules = get_subgraph_modules(graph, parent_graph)
```

Comment:
Do you mind explaining what we're changing in our graph partition here?

Reply:
The graph partition doesn't change, this change just collects all the modules used by this subgraph for use in onloading/offloading by the sequential pipeline.